easybuild/easyconfigs/g/GetFEM/GetFEM-5.4.1-foss-2019b-Python-3.7.4.eb

```diff
@@ -0,0 +1,59 @@
+# #
+# This is a contribution from HPCNow! (http://hpcnow.com)
+# Copyright::   HPCNow!
+# Authors::     Jordi Blasco <jordi.blasco@hpcnow.com>
+# License::     GPL-v3.0
+# #
+
+easyblock = 'ConfigureMake'
+
+name = 'GetFEM'
+version = '5.4.1'
+local_pyver = '3.7.4'
+versionsuffix = '-Python-%(local_pyver)s'
```

Comment:
Please remove the `local_pyver`, change `versionsuffix` back to normal, and add `Python` as a dependency.